Subject: remove Reqires.private from libcacard.pc
From: Michael Tokarev <mjt@tls.msk.ru>
Bug-Debian: http://bugs.debian.org/804289

Since we're not providing static library, Requires.private is not used.
But pkg-config complains about missing dependensies even if they're not
used.

--- a/libcacard.pc.in
+++ b/libcacard.pc.in
@@ -6,6 +6,5 @@ includedir=@includedir@
 Name: cacard
 Description: CA Card library
 Version: @PACKAGE_VERSION@
-Requires.private: nss glib-2.0
 Libs: -L${libdir} -lcacard
 Cflags: -I${includedir}/cacard
